docs/guide/apm-data-model.asciidoc

```diff
@@ -82,12 +85,17 @@ A transaction contains:
 
 * The timestamp of the event
 * A unique id, type, and name
-include::../context.asciidoc[]
+* Data about the environment in which the event is recorded:
+** Service - environment, framework, language, etc.
+** Host - architecture, hostname, IP, etc.
+** Process - args, PID, PPID, etc.
+** URL - full, domain, port, query, etc.
+** <<user-fields,User>> - (if supplied) email, ID, username, etc.
 * Other relevant information depending on the agent. Example: The JavaScript RUM agent captures transaction marks,
 which are points in time relative to the start of the transaction with some label.
 
-In addition, the agents provide some settings for users to capture customized information. This data is stored as not-indexed in a `custom` object.
-Searchable information is stored as `labels` instead.
+In addition, agents provide options for users to capture custom <<metadata, metadata>>.
+Metadata can be indexed - <<labels-fields,`labels`>>, or not-indexed - <<custom-fields,`custom`>>.
 
 TIP: Most agents limit keyword fields (e.g. `transaction.name`) to 1024 characters,
 and non-keyword fields (e.g. `labels`) to 10,000 characters.
```

Comment:
labels are keyword fields and should be limited to 1024 by agents. There are actually only a few fields which have the 10k char limit but that not yet completely aligned: https://github.com/elastic/apm-dev/issues/302#issuecomment-432570637

Reply:
Ahh, thanks for the clarification here